### PR TITLE
Login

### PR DESCRIPTION
**Pull request by juanlacu**
This PR contains:

* Login process completed.

### CODE REVIEW

File: src/styles/Common.scss [hunk 1]
```diff
@@ -51,3 +52,10 @@
 .margin-top-target {
   margin-top: 2.5rem;
 }
+
+.container-form {
+  max-width: 50rem;
+  width: 100%;
+
+  > * + * { margin-top: 2.5rem; }
```
Comment (MaicolBen) [inline src/styles/Common.scss:60]: Ugh, this is too generic, don't you have divs or something else to select?

Comment (juanlacu) [reply src/styles/Common.scss:60]: That was the idea, I have components, _h_ tags, _Links_, and others.

Comment (MaicolBen) [reply src/styles/Common.scss:60]: Got it

**Review by MaicolBen — APPROVED** [review]

File: src/styles/Common.scss [hunk 2]
```diff
@@ -7,6 +7,7 @@
   display: flex;
   flex-direction: column;
   align-items: center;
+  margin: 3rem;
```
Comment (Sebamax91) [inline src/styles/Common.scss]: Can you use `padding` instead? Because of how you are managing `box-sizing`.

**Pull request merged**